example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -69,6 +78,108 @@ async function setupCollectionAndTestData(config, testSuffix, testDataFolder) {
   ]);
 }
 
+/**
+ * Creates a new test collection with associated granule for testing.
+ *
+ * @param {string} prefix - stack Prefix
+ * @param {string} sourceBucket - testing source bucket
+ * @returns {Promise<Array>} A new collection with associated granule and a cleanup function to call after you are finished.
+ */
+const createActiveCollection = async (prefix, sourceBucket) => {
+  // The S3 path where granules will be ingested from
+  const sourcePath = `${prefix}/tmp/${randomId('test-')}`;
+
+  // Create the collection
+  const newCollection = await createCollection(
+    prefix,
+    {
+      duplicateHandling: 'error',
+      process: 'modis',
+    }
+  );
+
+  // Create the S3 provider
+  const provider = await createProvider(prefix, { host: sourceBucket });
+
+  // Stage the granule files to S3
+  const granFilename = `${randomId('junk-file-')}.txt`;
+  const granFileKey = `${sourcePath}/${granFilename}`;
+  await s3PutObject({
+    Bucket: sourceBucket,
+    Key: granFileKey,
+    Body: 'aoeu',
+  });
+
+  const granuleId = randomId('granule-id-');
+
+  const ingestTime = Date.now() - 1000 * 30;
+  const testExecutionId = randomId('test-execution-');
```

Comment:
You shouldn't need this since `buildAndExecuteWorkflow` will give you the ARN.

Reply:
1cc51a550 Hope this will pass the tests.